corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -306,6 +306,41 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
     });
 
+    const CaseTileGroupedView = CaseTileView.extend({
+        tagName: "div",
+        className: "formplayer-request list-cell-wrapper-style case-tile-group",
+        template: _.template($("#case-tile-grouped-view-item-template").html() || ""),
+        templateContext: function () {
+            const dict = CaseTileGroupedView.__super__.templateContext.apply(this, arguments);
+            dict['groupHeaderRows'] = this.options.groupHeaderRows;
+            dict['headerData'] = this.options.model.get('data').filter((_, i) => {
+                return this.options.headerRowIndices.includes(i);
+            });
+            dict['indexedRowDataList'] = this.getIndexedRowDataList();
+
+            return dict;
+        },
+
+        getIndexedRowDataList: function () {
```

Comment:
How feasible would it be to write a test for this?

Reply:
I spent a chunk of time trying to do this, but having a hard time with the set up.

I don't know if there's a way to access `getIndexedRowDataList` directly from `CaseTileGroupedView` but i'm trying to get it through first initializing `CaseTileGroupedListView` which requires initializing all the inherited views and that's giving me trouble.

Reply:
fair enough